src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { memo } from 'react';
+import cn from 'classnames';
+
+import { FilterStatus } from '../../types/FilterStatus';
+
+type Props = {
+  activeTodosAmount: number;
+  hasCompletedTodos: boolean;
+  filterType: FilterStatus;
+  onChangeType: React.Dispatch<React.SetStateAction<FilterStatus>>;
+  deleteCompletedTodos: () => void;
+};
+
+export const Footer: React.FC<Props> = memo((props) => {
+  const {
+    activeTodosAmount,
+    hasCompletedTodos,
+    filterType: filterStatus,
+    onChangeType,
+    deleteCompletedTodos,
+  } = props;
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${activeTodosAmount} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className={cn(
+            'filter__link',
+            { selected: filterStatus === FilterStatus.All },
+          )}
+          onClick={() => onChangeType(FilterStatus.All)}
+        >
+          All
+        </a>
+
+        <a
+          data-cy="FilterLinkActive"
+          href="#/active"
+          className={cn(
+            'filter__link',
+            { selected: filterStatus === FilterStatus.Active },
+          )}
+          onClick={() => onChangeType(FilterStatus.Active)}
+        >
+          Active
+        </a>
+
+        <a
+          data-cy="FilterLinkCompleted"
+          href="#/completed"
+          className={cn(
+            'filter__link',
+            { selected: filterStatus === FilterStatus.Completed },
+          )}
+          onClick={() => onChangeType(FilterStatus.Completed)}
+        >
+          Completed
+        </a>
+      </nav>
+
+      {hasCompletedTodos && (
```

Comment:
Note that when you remove the button in this way, the layout is eaten. Filtering should always be in the center of the footer. Try using a style (visibility-hidden) or a class (look for it in the bulm documentation)
![image](https://user-images.githubusercontent.com/102512764/216362416-feb851bb-bcdc-437d-a598-c63908d178ec.png)
![image](https://user-images.githubusercontent.com/102512764/216362563-cfaf85d8-d56e-431d-bdb9-e912d067ec20.png)

Reply:
fixed